.gitignore

```diff
@@ -11,8 +11,7 @@ Thumbs.db
 /node_modules/
 npm-debug.log
 /yarn.lock
+package-lock.json
```

Comment:
Shouldn't we commit the lock file?

Reply:
Libraries don't use lockfiles, you want the top-level project to resolve semver ranges to avoid massive duplication of deps in your node_modules tree.